libraries/AP_GPS/AP_GPS.h

```diff
@@ -142,6 +142,18 @@ class AP_GPS
         bool have_horizontal_accuracy:1;    ///< does GPS give horizontal position accuracy? Set to true only once available.
         bool have_vertical_accuracy:1;      ///< does GPS give vertical position accuracy? Set to true only once available.
         uint32_t last_gps_time_ms;          ///< the system time we got the last GPS timestamp, milliseconds
+
+        // all the following fields must only all be filled by RTK capable backend drivers
+        uint32_t rtk_time_week_ms;         ///< GPS Time of Week of last baseline in milliseconds
+        uint16_t rtk_week_number;          ///< GPS Week Number of last baseline
+        uint16_t rtk_age_cs;               ///< GPS age of last baseline correction in centiseconds  (0 when no corrections, 0xFFFF indicates overflow)
+        uint8_t  rtk_num_sats;             ///< Current number of satellites used for RTK calculation
+        uint8_t  rtk_baseline_coords_type; ///< Coordinate system of baseline. 0 == ECEF, 1 == NED
+        int32_t  rtk_baseline_x_mm;        ///< Current baseline in ECEF x or NED north component in mm
+        int32_t  rtk_baseline_y_mm;        ///< Current baseline in ECEF y or NED east component in mm
+        int32_t  rtk_baseline_z_mm;        ///< Current baseline in ECEF z or NED down component in mm
+        uint32_t rtk_accuracy;             ///< Current estimate of baseline accuracy (receiver dependent, typical 0 to 999.9)
```

Comment:
This needs a unit and dimension in the comments? My first guess would have been meters, but that wouldn't make sense in a uint32_t, and the 999.9 in the comments is unrepresentable in a uint32_t so I have no clue what unit this is expected to be in.

We also need to comment if this is considered the 2D or 3D accuracy? I'm guessing it's the 3D one, but without a comment that's an unfounded suspicion.

I realize these complaints are partly because the MAVLink spec wasn't clear enough, but we should do better on our internal documentation (and send a PR upstream to improve the comment).